README.md

```diff
@@ -22,3 +22,7 @@
 - [Kotlin lang](http://slack.kotlinlang.org/)
 - [ASP.NET Core](https://aspnetcore.slack.com/). Sign up [here](http://tattoocoder.com/aspnet-slack-sign-up/)
 - [Frontend Developers](http://frontenddevelopers.org/)
+- [Drupal](https://drupal.slack.com)
```

Comment:
@iAhmadZain plz add the link where the users can request getting an invitation for this community